ansible/configs/ocp4-cluster/destroy_env.yml

```diff
@@ -17,7 +17,7 @@
     - name: Remove DNS entry for OpenShift API and ingress
       nsupdate:
         server: "{{ osp_cluster_dns_server }}"
-        zone: "{{ osp_cluster_dns_zone }}"
+        zone: "{{ cluster_dns_zone }}"
```

Comment:
what is the point to proxy `osp_cluster_dns_zone` with `cluster_dns_zone` ?

Reply:
Continuity?

Reply:
Default it, I like the idea of having non-cloud provider vars where possible.

`
zone: "{{ cluster_dns_zone | default(osp_cluster_dns_zone) }}"
`